project/empty.go

```diff
@@ -94,3 +95,8 @@ func (e *EmptyService) Run(ctx context.Context, commandParts []string) (int, err
 func (e *EmptyService) RemoveImage(ctx context.Context, imageType options.ImageType) error {
 	return nil
 }
+
+// Events implements Service.Events but does nothing.
+func (e *EmptyService) Events(ctx context.Context, events chan eventtypes.Message) error {
```

Comment:
I'm pretty sure I'm going to forget to add new commands here in the future. Maybe we should have some sort of API test for `EmptyService` to verify it always implements `Service` :smile: